### ISSUE TITLE
Accessibility: edit menu item non-link and title attribute

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#93]
The edit menu item should be a button of course, with an aria-expanded attribute.
Additionally, having a title attribute that exactly repeats the text is completely redundant:
```
<a class="item-edit" id="edit-1760" title="Edit Menu Item" href="#">Edit Menu Item</a>
```
Title attribute should be removed and used just in very rare cases. See core ticket https://core.trac.wordpress.org/ticket/24766 and the several related tickets.

Consider to add a better descriptive text: "edit menu item... ok but which item? See techniques suggested for similar cases.

**Comment by afercia** [comment]
To clarify, here's how Firefox+NVDA will read out the edit menu items thing when tabbing through items:
```
list with 1 items
clickable
Edit Menu Item
visited  link  Edit Menu Item
```
- "list with 1 items" because each item is a DL. The whole list should be one single list, this way screen readers will correctly announce the number of items.
- "clickable" because the `dt` has a click handler attached
- the actual item title doesn't get read out because it's in a non-focusable span
- "Edit Menu Item" it's a focusable link (should be a button)
- "Edit Menu Item" (again) redundant title attribute
